src/aiida_quantumespresso/calculations/ph.py

```diff
@@ -72,6 +72,11 @@ def define(cls, spec):
             message='The stdout output file was incomplete probably because the calculation got interrupted.')
         spec.exit_code(350, 'ERROR_UNEXPECTED_PARSER_EXCEPTION',
             message='The parser raised an unexpected exception: {exception}')
+        spec.exit_code(360, 'ERROR_INCOMPATIBLE_FFT_GRID',
+            message='The FFT grid is incompatible with the detected symmetries. Try using the correct ibrav.')
```

Comment:
Maybe we should explicitly state that the `ibrav` should be set in the parent `pw.x`. I also wasn't sure if we should use "correct" here:

```suggestion
            message='The FFT grid is incompatible with the detected symmetries. Try using the lattice-specific `ibrav` != 0 in the parent `pw.x` calculation.')
```

Do you know if these errors only happen when `ibrav == 0`?

Same for the other exit code message.

Reply:
I still didn't try, but reading on the QE documentation it seems that by specifying ibrav the error should solve (it is really due to how QE-ph handles symmetries)

Reply:
Alright, let's leave it like this for now. In case we ever see that the error pops up for `ibrav != 0` we'll adapt.